example/src/app.js

```diff
@@ -21,7 +21,5 @@ export const App = (props, { state, effects }) => {
   );
 };
 
-App.contextTypes = contextTypes;
 
-
-export default addState(App);
+export default addState(injectState(App));
```

Comment:
Hmm that's a good point.  I've been wanting to rename `withState` anyway; thinking through all of the names is def worth doing.